py3status/modules/README.md

```diff
@@ -2722,6 +2724,297 @@ Requires:
 
 ---
 
+### <a name="weather_owm"></a>weather_owm
+
+Ultimately customizable weather module based on the
+[IP-API Geolocation API](http://ip-api.com) and the
+[OpenWeatherMap API](https://openweathermap.org).
+
+Requires an API key for OpenWeatherMap (OWM), but the
+[free tier](https://openweathermap.org/price) allows you enough
+requests/sec to get accurate weather even up to the second.
+
+Note that this module does require an additional python package,
+[pyowm](https://github.com/csparpa/pyowm), and this may be easily
+installed via `pip`.
+
+This module allows you to specify an icon for nearly every weather scenario
+imaginable. The default configuration options lump many of the icons into
+a few groups, and due to the limitations of UTF-8, this is really as expressive
+as it gets.
+
+I would highly suggest you install an additional font, such as the incredible
+(and free!) [Weather Icons font](https://erikflowers.github.io/weather-icons),
+which has icons for most weather scenarios.
+
+For more information, see
+[the documentation](https://openweathermap.org/weather-conditions)
+on what weather conditions are supported. See the configuration options
+for how to specify each weather icon.
+
+Configuration parameters:
+  - `api_key`: Your [OpenWeatherMap API key](https://openweathermap.org/appid)
+    *(default None)*
+
+  - `lang`: A two-letter code for your language.
+    See [supported codes](https://openweathermap.org/current#multi)
+    *(default 'en')*
+
+  - `req_timeout`: The timeout in seconds for contacting the IP API.
+    *(default 10)*
+
+  - `cache_duration`: The time between weather polling in seconds
+    It is recommended to keep this at a higher value to avoid rate
+    limiting with the API's.
+    *(default 60)*
```

Comment:
Site asks not to send requests more than 1 time per 10 min.
Normally the weather is not changing so frequently. Use `600`~?
└── If too many i3/py3status restarts, API could get blocked/locked out.

Source: https://openweathermap.org/appid